src/core/Table/TableCell.tsx

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { Cell, CellProps, TableInstance } from 'react-table';
+import cx from 'classnames';
+import { getCellStyle } from './utils';
+import EditableCell, { OnCellEditCallbackParams } from './EditableCell';
+import SubRowExpander from './SubRowExpander';
+import { SELECTION_CELL_ID } from './hooks';
+
+export type TableCellProps<T extends Record<string, unknown>> = {
+  cell: Cell<T>;
+  cellIndex: number;
+  onCellEdit?: (cellEditParams: OnCellEditCallbackParams<T>) => void;
+  isRowDisabled: boolean;
+  tableHasSubRows: boolean;
+  tableInstance: TableInstance<T>;
+  expanderCell?: (cellProps: CellProps<T>) => React.ReactNode;
```

Comment:
This is not really a "cell" if it's part of another cell.

Reply:
Just left the same naming as it comes from the `Table` 🤷‍♂️